public/js/delete-video.js

```diff
@@ -0,0 +1,31 @@
+window.addEventListener('load', init)
+
+function init() {
+    let links = document.querySelectorAll("[data-delete-video]")
+    for (let link of links) {
+        link.addEventListener("click", function (e) {
+            e.preventDefault()
+            let eArray = e.composedPath()
+
+            if (confirm("Voulez-vous supprimer cette video ?")) {
+                fetch(this.getAttribute("href"), {
+                    method: "DELETE",
+                    headers: {
+                        "X-Requested-With": "XMLHttpRequest",
+                        "Content-type": "application/json"
+                    },
+                    body: JSON.stringify({"_token": this.dataset.token})
+                }).then(response => response.json()
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/VinzOo93/P6SnowTricks/pullRequest?prid=8761575)
```suggestion
                }).then((response) => response.json()
```